src/main/scala/com/campudus/tableaux/database/model/TableauxModel.scala

```diff
@@ -101,61 +123,102 @@ class TableauxModel(override protected[this] val connection: DatabaseConnection)
     }
   }
 
+  def updateValue[A](tableId: TableId, columnId: ColumnId, rowId: RowId, value: A): Future[Cell[_, _]] = for {
+  // TODO column should be passed on to insert methods
```

Comment:
What is meant by this? Is it possible to open an issue, elaborate on it and get rid of the TODO comments?